.gitignore

```diff
@@ -4,3 +4,5 @@
 .venv
 .sconsign.dblite
 build
+
+.vscode
```

Comment:
Can we have these sorted in ASCII order?

Reply:
I'll leave a comment; it's in a separate section because the preceding lines are for autogenerated/build stuff. `.vscode` is for the developer.